components/rmrk/Gallery/History.vue

```diff
@@ -118,7 +118,7 @@
 import { urlBuilderBlockNumber } from '@/utils/explorerGuide'
 import ChainMixin from '@/utils/mixins/chainMixin'
 import { Component, Prop, Watch, mixins } from 'nuxt-property-decorator'
-import { Interaction } from '../service/scheme'
+import { Interaction as EventInteraction } from '../service/scheme'
```

Comment:
just FYI: I call it `Event` in the rubick but it could be ambiguous in the JS context

Reply:
Got it. Should I change the nickname of `EventInteraction`?

Reply:
I think it's fine as it is :)

Reply:
OK